app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -33,15 +33,28 @@ def update
       end
     end
 
+    def self.can_do_hybrid(session:)
```

Comment:
So we want to prevent users from being able to get around the How To Verify page, so we need to set a precondition for each page that follows the How To Verify Page. This method does a quick config check and if it passes, requires that the `skip_doc_auth` session value is a boolean. If the config check fails, we allow the `skip_doc_auth` value to be nil or false. The else handles the case where IPP is not enabled.